docs/modules/fault-tolerance/pages/backups.adoc

```diff
@@ -7,6 +7,15 @@ The distribution happens on the partition level; the data and its backups are st
 memory partitions. See the xref:overview:data-partitioning.adoc[Data Partitioning] and xref:clusters:partition-group-configuration.adoc[Partition Grouping]
 for more information about the partitioning.
 
+[IMPORTANT]
+====
+Hazelcast does not support backup redistribution on a data structure level. Thus, changing the backup count of a nonempty data structure is **not** supported. For example, for a map with a nonzero backup count:
+
+1. increasing the backup countfootnote:change-backup-count[The backup count of a map can only be changed by modifying the xref:configuration:understanding-configuration.adoc#static-configuration[static configuration] and restarting the cluster.] does **not** create additional copies of existing map entries,
+2. decreasing the backup countfootnote:change-backup-count[] does **not** remove any backups of existing map entries, so some backups will have stale data if the map entries are overwritten, and
```

Comment:
`countfootnote:change-backup-count[]`

the adoc syntax here is crazy, but surprisingly it works. Will it work also with extra space before `footnote:`?

Reply:
Unfortunately, the extra space becomes part of the document, and [the workaround][1] suggested in the AsciiDoc documentation is ugly.

[1]: https://docs.asciidoctor.org/asciidoc/latest/macros/footnote#:~:text=If%20you%20find,%29.

Reply:
meh